pkg/helm/helmpath/helmhome_unix_test.go

```diff
@@ -36,6 +36,7 @@ func TestHelmHome(t *testing.T) {
 	isEq(t, hh.LocalRepository(), "/r/repository/local")
 	isEq(t, hh.Cache(), "/r/repository/cache")
 	isEq(t, hh.CacheIndex("t"), "/r/repository/cache/t-index.yaml")
+	isEq(t, hh.RelativeIndex("t"), "t-index.yaml")
```

Comment:
hmm... Wouldn't we want the relative path be equal to `repository/cache/t-index.yaml`, as in relative to the home directory's path?

Reply:
It'd make sense when relative to repositories.yaml, but not the home directory.

It's all relative :)

Reply:
it has to be relative to `hh.Cache()`, due to https://github.com/kubernetes/helm/blob/8cf72adf521311225274fa835a2b04060eeb781a/pkg/repo/chartrepo.go#L136-L138